interface/billing/get_claim_file.php

```diff
@@ -47,6 +91,14 @@
 
     // dump the picture and stop the script
     fpassthru($fp);
+
+    // If the caller sets the delete flag, delete the file when we're done serving it
+    // This is the common case of a temporary file when validation-only is performed
+    // by the BillingProcessor
+    if (isset($_GET['delete']) &&
+        $_GET['delete'] == 1) {
```

Comment:
if (
        isset($_GET['delete']) &&
        $_GET['delete'] == 1

Reply:
usually have my squiz outside of the repo and then can run this
```
~/.composer/vendor/squizlabs/php_codesniffer/bin/phpcbf interface/billing/get_claim_file.php

Reply:
Thanks @stephenwaite I ran this on src/Billing/X125010837P.php ... we'll see if the CI likes it! Thanks for the tip, cause there were like 200 detected errors in that file!